tests/test_parameters.py

```diff
@@ -0,0 +1,165 @@
+"""Tests of ParameterGenerator() methods involving drawing parameters."""
+import psfws
+import numpy as np
+import pandas as pd
+
+
+def dict_test_helper(thing, p_list, x):
+    """Test whether all keys in p_list exist in p_list and checks lengths."""
+    # this loop starts with testing x for later comparison
+    np.testing.assert_equal(x in thing.keys(), True,
+                                err_msg=f'{x} not in dict!')
```

Comment:
For this, you should probably just use a regular assert, not assert_equal:
```
assert x in test_dict.keys(), f'{x} not in dict!'
```
Lots of other cases below like this, which are probably clearer with a regular assert.  The main use of numpy's `assert_equal` is for arrays, since arr1 == arr2 returns a boolean array, not a single boolean saying whether the arrays are equal.

Reply:
I've removed this one instance (see above) but have changed the assert_equals following that I think are eligible cases.